### ISSUE TITLE
Possible error on slide 69.

### ISSUE THREAD

**Issue by cogle** [vinniefalco/CppCon2017#1]
Hello Vinnie,

I would just like to confirm should slide number 69 be

`res.body() = “C:\\Users\Vinnie\www\index.html”;`

Instead of 

`res.body = “C:\\Users\Vinnie\www\index.html”;`

**Comment by roger-tannous** [comment]
No, it must be res.body and not res.body() because just before Vinnie tackled sending a file contents in the response, he said that we're just gonna continue with body as data member.
